sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/data/DoublePointData.java

```diff
@@ -16,9 +18,38 @@
 @Immutable
 @AutoValue
 public abstract class DoublePointData implements PointData {
+
+  /**
+   * Creates a {@link DoublePointData}.
+   *
+   * @param startEpochNanos (optional) The starting time for the period where this point was
```

Comment:
Is this really optional? If so we need to document something like `0 indicates no starting time.`

Reply:
According to the data model (which this is copy-pasted from) yes.

From and SDK standpoint, we should never export a point without a starting time from our own instruments.   The only place we expect this to happen is in the collector (when adapting other metric formats).   I can update this with that nuance.